domains/measurements/Measurements.API/Tests/HealthControllerTests.cs

```diff
@@ -3,7 +3,7 @@
 using Microsoft.AspNetCore.Mvc.Testing;
 using Xunit;
 
-namespace Integration.Tests.Controllers;
+namespace Tests;
```

Comment:
Why this change?

Reply:
Because the folder is named "Tests" and is reciding in the Measurements.API/Tests/ folder. I believe this project, hasn't been updated in a LONG time.